app/src/androidTest/java/ch/epfl/sdp/peakar/user/ProfileActivityTest.java

```diff
@@ -47,7 +47,7 @@
 import static org.junit.Assert.assertSame;
 import static org.junit.Assert.fail;
 
-public class ProfileActivityTest {
+public class    ProfileActivityTest {
```

Comment:
```suggestion
public class ProfileActivityTest {
```
I think you accidentally added some blank spaces here.